source/_docs/autostart/systemd.markdown

```diff
@@ -51,7 +51,7 @@ After=network-online.target
 [Service]
 Type=simple
 User=%i
-ExecStart=/srv/homeassistant/bin/hass -c "/home/homeassistant/.homeassistant"
+ExecStart=/srv/homeassistant/bin/hass -c "/home/[your user]/.homeassistant"
```

Comment:
We generally use an all-caps single string for replacement values in our documentation. e.g., `YOUR_USER`.

```suggestion
ExecStart=/srv/homeassistant/bin/hass -c "/home/YOUR_USER/.homeassistant"
```

Reply:
Sounds plausible but on the same page [your user] is used everywhere else. Maybe all occurences of [your user] should be changed then.

Reply:
Done for this page!